libmamba/CMakeLists.txt

```diff
@@ -552,23 +543,13 @@ if (BUILD_STATIC)
     # On Windows, a static library should use a different output name
     # to avoid the conflict with the import library of a shared one.
     if (CMAKE_HOST_WIN32)
-        libmamba_create_target(libmamba-static STATIC SHARED libmamba_static)
-    else ()
-        libmamba_create_target(libmamba-static STATIC SHARED libmamba)
-    endif ()
-endif ()
-
-if (BUILD_STATIC_DEPS)
-    message(STATUS "Adding full-static libmamba target")
-
-    if (CMAKE_HOST_WIN32)
-        libmamba_create_target(libmamba-full-static STATIC STATIC libmamba_full_static)
+        libmamba_create_target(libmamba-static STATIC libmamba_static)
     else ()
-        libmamba_create_target(libmamba-full-static STATIC STATIC libmamba_full_static)
+        libmamba_create_target(libmamba-static STATIC libmamba)
     endif ()
 endif ()
 
-if (NOT (BUILD_SHARED OR BUILD_STATIC OR BUILD_STATIC_DEPS))
+if (NOT (BUILD_SHARED OR BUILD_STATIC))
```

Comment:
Should there be s single option, where true means static, false means shared?

Reply:
yeah I think we should default to `SHARED`

Reply:
I think `BUILD_SHARED` and `BUILD_STATIC` are more expressive. And you may want to build both at the same time.

Reply:
Ha right, it's too target, not one that transmute

Reply:
> yeah I think we should default to `SHARED`

In that case if you want to build the static version only, you need to pass two options ( `BUILD_SHARED=OFF`  and `BUILD_STATIC=ON`) instead of a single one

Reply:
I can add options like:
- BUILD_ALL_SHARED
- BUILD_ALL_STATIC

The only question is should we build tests with these targets? (sometimes you just want to build micromamba for reproducing an issue or debugging, and you don't want tot build the tests).